src/styles/blocks/product.scss

```diff
@@ -0,0 +1,120 @@
+.product-card {
+  box-sizing: border-box;
+
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+
+  width: $product-card-width;
```

Comment:
Ensure that the SCSS variable `$product-card-width` is defined in your project. If it's not defined, this will cause a compilation error.